### PR TITLE
Molecules Stories

### PR DESCRIPTION
**Pull request by NKoech123**
## Linear Ticket
PRO-304
https://linear.app/govrn/issue/PRO-304/stories-for-the-component-library-molecules

## Description
Creating Stories for molecules.

# Stories with no errors:
Screen Shot 2022-08-02 at 12 15 23 PM
Screen Shot 2022-08-02 at 12 15 39 PM

# Stories with errors:
Screen Shot 2022-08-02 at 12 17 22 PM
Screen Shot 2022-08-02 at 12 17 38 PM
Screen Shot 2022-08-02 at 12 17 53 PM
Screen Shot 2022-08-02 at 12 18 04 PM

## Screencaptures
Dropping them next day. Trying to get passed chakraUI issues, again

### CODE REVIEW

**Comment by jonathanprozzi** [comment]
@NKoech123 which Chakra issues are you facing with this? I think we can set this to a Draft PR if it's not ready yet

**Comment by jonathanprozzi** [comment]
Thanks for adding the screenshots! I'll take a look -- also it seems like the global styles for the components aren't loading (the inputs look to be using raw HTML instead of our styled ones)

File: libs/protocol-ui/src/stories/molecules/DatePicker.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Story } from '@storybook/react';
+import { DatePicker, DatePickerProps } from '../..';
+
+export default {
+    title: 'Components/Molecules/DatePciker',
```
Comment (jonathanprozzi) [inline libs/protocol-ui/src/stories/molecules/DatePicker.stories.tsx:6]: small typo: 
`title: 'Components/Molecules/DatePciker',` -> `title: 'Components/Molecules/DatePicker',`

File: libs/protocol-ui/src/stories/molecules/DatePicker.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    component: DatePicker,
+  
+}
+
+const Template: Story<DatePickerProps> = (args) => <DatePicker {...args} />;
+
+
+export const Default = Template.bind({});
+
+Default.args = {
```
Comment (jonathanprozzi) [inline libs/protocol-ui/src/stories/molecules/DatePicker.stories.tsx:16]: Let's try to use an actual example -- can base off our actual DatePicker in the frontend app

**Comment by jonathanprozzi** [comment]
@NKoech123 @alexkeating 
Caught the branch up to `staging` and fixed the Stories issues, although the git diff is pretty massive. I'm thinking this is because I merged in staging after the prettierrc style change? Pretty bizarre history and wasn't sure why

**Comment by alexkeating** [comment]
It should be merged now, squashed to make the diff readable

**Pull request closed without merging**